python/requirements_compiled.txt

```diff
@@ -1519,7 +1519,7 @@ py-cpuinfo==9.0.0
     # via deepspeed
 py-partiql-parser==0.5.0
     # via moto
-py-spy==0.3.14
+py-spy==0.4.0
```

Comment:
oh, you can upgrade this in python/requirements.txt files i think

Reply:
you can copy the generated content for this PR here https://ray-ci-artifact-pr-public.s3.amazonaws.com/bc9fbdaa9cc5d971bca51db672e7692b91dc316a/tmp/artifacts/requirements_compiled.txt